lib/test_prof/recipes/rspec/let_it_be.rb

```diff
@@ -32,6 +34,19 @@ def register_modifier(key, on: :let, &block)
         LetItBe.modifiers[key] = Modifier.new(on, block)
       end
 
+      def report_duplicates=(value)
+        value = value.to_sym
+        unless %i[warn raise].include?(value)
+          raise ArgumentError, "#{value} is not acceptable, acceptable values are :warn or :raise"
+        end
+
+        @report_duplicates = value
+      end
+
+      def report_duplicates
+        @report_duplicates ||= :warn
```

Comment:
I think, the default should be false meaning that we don't need any warnings (current behaviour).